IonDotnet/Internals/Text/TextScanner.cs

```diff
@@ -1976,11 +2009,24 @@ public int LoadTripleQuotedString(StringBuilder sb, bool isClob)
                     //                        c = Characters.GetLowSurrogate(c);
                     //                    }
                 }
+                else
+                {
+                    // CLOB texts should be only 7-bit ASCII characters
+                    Required7BitChar(c);
+                }
 
                 sb.Append((char)c);
             }
         }
 
+        private void Required7BitChar(int c)
```

Comment:
This is minor, but I recommend renaming this `Require7BitChar` so that it reads as an action like the other methods in this class.